services/auth/oauth2.go

```diff
@@ -120,10 +122,10 @@ func (o *OAuth2) userIDFromToken(ctx context.Context, tokenSHA string, store Dat
 		}
 
 		// Otherwise, check if this is an OAuth access token
-		uid := CheckOAuthAccessToken(ctx, tokenSHA)
+    accessTokenScope, uid := GetOAuthAccessTokenScopeAndUserID(ctx, tokenSHA)
```

Comment:
```suggestion
		accessTokenScope, uid := GetOAuthAccessTokenScopeAndUserID(ctx, tokenSHA)
```

fix merge and conflict